miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridgeDispatcher.kt

```diff
@@ -1,26 +1,13 @@
 package com.rakuten.tech.mobile.miniapp.js.userinfo
 
-import androidx.annotation.VisibleForTesting
-import com.google.gson.Gson
 import com.rakuten.tech.mobile.miniapp.MiniAppSdkException
-import com.rakuten.tech.mobile.miniapp.js.MiniAppBridgeExecutor
-import com.rakuten.tech.mobile.miniapp.permission.MiniAppCustomPermissionCache
-import com.rakuten.tech.mobile.miniapp.permission.MiniAppCustomPermissionType
+import com.rakuten.tech.mobile.miniapp.js.ErrorBridgeMessage.NO_IMPL
 import java.util.ArrayList
 
 /**
  * A class to provide the interfaces for getting user info e.g. user-name, profile-photo etc.
  */
-@Suppress(
-    "TooGenericExceptionCaught", "LongMethod", "UnnecessaryAbstractClass",
-    "LargeClass", "TooManyFunctions"
-)
-abstract class UserInfoBridgeDispatcher {
-
-    private lateinit var bridgeExecutor: MiniAppBridgeExecutor
-    private lateinit var customPermissionCache: MiniAppCustomPermissionCache
-    private lateinit var miniAppId: String
-
+interface UserInfoBridgeDispatcher {
```

Comment:
I think, this a breaking change for the host app because now they need to call it as `object : UserInfoBridgeDispatcher`

Reply:
So unfortunate that we used abstract class before but it's okay, I will bring it back as abstract class to not create a change in host app.